lib/howl/ui/editor.moon

```diff
@@ -306,11 +306,12 @@ class Editor extends PropertyObject
     @buffer\as_one_undo -> f lines
 
   with_position_restored: (f) =>
-    line, column, indentation = @cursor.line, @cursor.column, @current_line.indentation
+    line, column, indentation, top_line = @cursor.line, @cursor.column, @current_line.indentation, @line_at_top
     status, ret = pcall f, self
     @cursor.line = line
     delta = @current_line.indentation - indentation
     @cursor.column = column + delta
+    @line_at_top = top_line
```

Comment:
Ok, leave it in for now. Yes, the drawing will be done once but display calculations to move the page could happen twice (and will definitely aullar).